Alignment/OfflineValidation/python/TkAlAllInOneTool/zMuMuValidationTemplates.py

```diff
@@ -23,11 +23,11 @@
 ########### standard includes ##############################
 process.load("Configuration.StandardSequences..oO[magneticField]Oo._cff")
 process.load("Configuration.StandardSequences.Reconstruction_cff")
-process.load("Configuration.Geometry.GeometryDB_cff")
+process.load("Configuration.Geometry.GeometryRecoDB_cff")
 
 
 ########### DATABASE conditions ############################
-process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```

Comment:
@hroskes in 76X and 75X V2 is default, in 74X you should keep the additional part in the name.